apps/public-docsite-v9/src/Concepts/Upgrade/FromV8/Components/Button.stories.mdx

```diff
@@ -0,0 +1,74 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Concepts/Upgrading/from v8/Components/Button Upgrade" />
+
+# Button Upgrade
+
+The v8 button variants share the IButtonProps. When using ButtonBase the resulting variant is rendered based on the buttonType (normal, primary, hero, compound, command, icon, or default) and when using a variant directly then props like primary and split change how the button is rendered.
+
+In v9, there is not a shared button type. Variants have their own props although they may extend ButtonProps. There are not variants for primary and action buttons. This is controlled through the appearance prop.
+
+To upgrade a button replace the v8 usage with the corresponding v9 button variant. Use the property mapping to pass the right properties to the v9 button.
+
+## Button Variant Mapping
+
+| v8 Button variant    | v9 Button variant               |
+| -------------------- | ------------------------------- |
+| DefaultButton        | Button                          |
+| ActionButton         | Button appearance="transparent" |
+| CommandButton        | Button                          |
+| CommandBarButton     | Button                          |
+| CompoundButton       | CompoundButton                  |
+| ContextualMenuButton | Menu / MenuButton               |
+| PrimaryButton        | Button appearance="primary      |
+| Button split         | SplitButton                     |
+| Button toggle        | ToggleButton                    |
+
+## Button Props Mapping
+
+Properties with Y in the Shim column are handled when using the button shims.
```

Comment:
What do you mean by "properties with Y in the Shim column"? I don't see any "Y"s there.